Content.Client/Entry/IgnoredComponents.cs

```diff
@@ -300,6 +300,7 @@ public static class IgnoredComponents
             "Artifact",
             "RandomArtifactSprite",
             "EnergySword",
+            "Bible"
```

Comment:
FYI you'll get conflicts if you dump it at the end, especially without a trailing comma, just add it randomly in the middle.